Lib/test/test_faulthandler.py

```diff
@@ -578,7 +578,7 @@ def run(self):
         regex = r"""
             ^Thread 0x[0-9a-f]+ \(most recent call first\):
             (?:  File ".*threading.py", line [0-9]+ in [_a-z]+
-            ){{1,3}}  File "<string>", line 23 in run
+            ){{1,3}}  File "<string>", line (?:22|23) in run
```

Comment:
It might a bit puzzling to future readers... Perhaps we should add a short comment about this above the regex?